frontend/package.json

```diff
@@ -34,6 +35,7 @@
     "@angular/platform-browser-dynamic": "^8.2.14",
     "@angular/router": "^8.2.14",
     "@sentry/browser": "^5.6.3",
+    "js-marker-clusterer": "^1.0.0",
```

Comment:
Just curious, do we need this package as well for @agm/js-marker-clusterer or is it a leftover from your prototyping? Doesn't the @agm/js-marker-clusterer manage it's own dependencies?

Reply:
It's needed as peer dependency. It's mentioned in the README.md and it fails without this dependency. I was also a bit surprised.
https://github.com/SebastianM/angular-google-maps/tree/master/packages/js-marker-clusterer#installation